tasks/component-reporter.js

```diff
@@ -138,7 +138,6 @@ async function main({
 	componentName = process.env.NX_TASK_TARGET_PROJECT,
 	cwd,
 } = {}) {
-	console.log("componentName", componentName);
```

Comment:
I snuck this fix in - it was a debugging comment I was using earlier and didn't mean for it to go into production.